src/maintainer/knowledge_base.rst

```diff
@@ -270,6 +270,27 @@ A package that needs all five compilers would define
 
 .. _cdt_packages:
 
+Activation recipes
+
+  1. GCC - https://github.com/conda-forge/ctng-compiler-activation-feedstock
+  2. Clang, OSX - https://github.com/conda-forge/clang-compiler-activation-feedstock
+  3. gfortran, OSX - https://github.com/conda-forge/gfortran_osx-64-feedstock
+
+Implementation packages
+
+  1. GCC - https://github.com/conda-forge/ctng-compilers-feedstock
+
+  2. gfortran, OSX - https://github.com/conda-forge/gfortran_impl_osx-64-feedstock
+
+  3. clang
+    https://github.com/conda-forge/llvmdev-feedstock
+    https://github.com/conda-forge/libcxx-feedstock
+    https://github.com/conda-forge/clangdev-feedstock
+    https://github.com/conda-forge/openmp-feedstock
+    https://github.com/conda-forge/tapi-feedstock
+
+  4. OSX - https://github.com/conda-forge/cctools-and-ld64-feedstock
+
```

Comment:
Is this block part of the PR?

Reply:
Yeah, this seems like something that was accidentally committed to the wrong branch.